clients/lsp-lua.el

```diff
@@ -592,6 +592,73 @@ and `../lib` ,exclude `../lib/temp`.
   :priority -3
   :server-id 'lsp-lua-lsp))
 
+;;; lua-roblox-language-server
+(defgroup lsp-roblox-lua-language-server nil
+  "Roblox Lua LSP client, provided by the Roblox Lua Language Server."
+  :group 'lsp-mode
+  :version "7.1"
+  :link '(url-link "https://github.com/NightrainsRbx/RobloxLsp"))
+
+(defcustom lsp-clients-roblox-lua-language-server-install-dir (f-join lsp-server-install-dir "roblox-lua-language-server/")
```

Comment:
nit: the `lsp-clients` prefix is the relic from previous `lsp-clients` which has been refactored out.
You can skip the `clients` part here and use `lsp-roblox` directly (or use `lsp-lua-roblox` as prefix since the file name is `lsp-lua.el` already)